packages/amplify-category-function/src/provider-utils/awscloudformation/utils/cronExpression.ts

```diff
@@ -0,0 +1,641 @@
+/*
+AWS Cron Expression generator
+Code Taken from : https://code.amazon.com/packages/JetstreamUtils/blobs/mainline/--/src/com/amazon/jetstream/model/schedule/CronExpression.java
```

Comment:
not sure what the policy on exposing a link like this publicly is. I know only people on corpnet can access it, but maybe it's still a no-no?

Reply:
no-no, remove the whole comment